xbmc/GUIInfoManager.cpp

```diff
@@ -4043,6 +4046,13 @@ std::string CGUIInfoManager::GetVideoLabel(int item)
         return strRatingAndVotes;
       }
       break;
+    case VIDEOPLAYER_USER_RATING:
+    {
+      std::string strUserRating;
+      if (m_currentFile->GetVideoInfoTag()->m_iUserRating > 0)
+        strUserRating = StringUtils::Format("%i", m_currentFile->GetVideoInfoTag()->m_iUserRating);;
```

Comment:
The whole file is crawling with them, not sure if it's okay if I fix them inside this PR? Left them out on purpose, but must have copied somewhere wrong.